backend/resources/emails/change-email/en.html

```diff
@@ -450,7 +450,7 @@
                 <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:top;" width="100%">
                   <tr>
                     <td align="center" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                      <div style="font-family:Source Sans Pro, sans-serif;font-size:14px;line-height:150%;text-align:center;color:#64666A;">Penpot © 2020 | Made with &lt;3 and Open Source</div>
+                      <div style="font-family:Source Sans Pro, sans-serif;font-size:14px;line-height:150%;text-align:center;color:#64666A;">Penpot @ 2021 | Made with &lt;3 and Open Source</div>
```

Comment:
Wouldn't be necessary to edit the mjml files instead? And then perhaps execute `./scripts/build-email-templates.sh`

Reply:
If you don't change mjml files, next time the script is executed, this will be reset to 2020.

Reply:
We'll see it later. For the moment I merge this.